topsort-api-v2.yml

```diff
@@ -117,7 +117,7 @@ paths:
                       location: New York
                   - type: banners
                     slots: 1
-                    aspectRatio: '4:1'
+                    slotId: 254b6dfc-3447-4ddf-a270-ed23101023ef
```

Comment:
Are slot-ids going to be uuids? That could be error prone for developers.

Reply:
Not necessarily, it will be provided by the marketplace, so it could be a UUID or a number or anything really

Reply:
Changed to a kebab case string.